Digipost.Signature.Api.Client.Core/PersonalIdentificationNumber.cs

```diff
@@ -1,36 +1,16 @@
 ﻿namespace Digipost.Signature.Api.Client.Core
 {
-    public class PersonalIdentificationNumber
+    public class PersonalIdentificationNumber : Identifier
     {
-        public PersonalIdentificationNumber(string value)
+        public PersonalIdentificationNumber(string value) : base(value)
         {
-            Value = value;
         }
 
-        public string Value { get; }
-
         private string MaskedValue => Value.Substring(0, 6) + "*****";
 
         public override string ToString()
         {
-            return $"{MaskedValue}";
-        }
-
-        public override bool Equals(object obj)
-        {
-            var other = obj as PersonalIdentificationNumber;
-
-            return other != null && string.Equals(other.Value, Value);
-        }
-
-        public override int GetHashCode()
-        {
-            unchecked
-            {
-                var hash = 17;
-                hash = hash * 23 + Value.GetHashCode();
-                return hash;
-            }
+            return $"{nameof(MaskedValue)}: {MaskedValue}";
```

Comment:
Human readable format på toStringen, ala «personal identification number of 123456*****»